### PR TITLE
fix: make changes to run bolt weekly jobs

### PR DESCRIPTION
**Pull request by blaisedias**
Refactor to allow ubuntu LTS runs with different testplans for
 - bolt
 - mayastor

Fix weekly pipline jobs to use testplans
	- MQ-2743 for bolt
	- MQ-2864 for mayastor (new)

### CODE REVIEW

**Review by rohan2794 — APPROVED** [review]
To maintain consistency , mayastor Jenkinsfile should be under mayastor directory similar to bolt

File: pipelines/bolt/weekly/Jenkinsfile [hunk 1]
```diff
@@ -8,6 +8,12 @@ def Common() {
 
 def image_tag = ""
 def datacore_bolt = true
+def testplan = 'MQ-2743'
+def job_params = [
```
Comment (chriswldenyer) [inline pipelines/bolt/weekly/Jenkinsfile:12]: nit: call this test_job_params to clarify that these are used for the test stages vs build stages.

**Review by chriswldenyer — APPROVED** [review]
LGTM, one suggestion.
I second Rohan's suggestion about using a directory for mayastor.

File: pipelines/bolt/weekly/Jenkinsfile (hunk 1 above)
Comment (blaisedias) [reply pipelines/bolt/weekly/Jenkinsfile]: fixed https://github.com/mayadata-io/mayastor-e2e/pull/475/commits/cee64eed95ce5c6bb8e57cbbc2cbfea7975d9f68

**Pull request merged**